lang/build.gradle

```diff
@@ -48,17 +47,12 @@ dependencies {
     api 'org.partiql:partiql-ir-generator-runtime:0.3.0'
 
     // test-time dependencies
-    testImplementation 'org.jetbrains.kotlin:kotlin-reflect'
     testImplementation 'org.jetbrains.kotlin:kotlin-test-junit5'
     testImplementation 'pl.pragmatists:JUnitParams:[1.0.0,1.1.0)'
     testImplementation 'org.assertj:assertj-core:[3.11.0,3.12.0)'
 
-    testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.7.0'
```

Comment:
We need this dependency to run  junit `ParameterizedTest`s with `ArgumentsSource` that we are starting to use (e.g. [EvaluatingCompilerFromLetTests.kt](https://github.com/partiql/partiql-lang-kotlin/blob/0ca86ac64984350dd236552de99519f2115a9286/lang/test/org/partiql/lang/eval/EvaluatingCompilerFromLetTests.kt)). Without this dependency, these tests aren't being run by gradle.

Reply:
Oops nice catch thanks! This is probably why I should not make wholesale changes without understanding what I am actually doing :)